sag_et.par

```diff
@@ -4,195 +4,241 @@
 ! s = list of subdivisions for above division using non-native fifth
 !
 n 5 3
+/||\
 n 7 4
+/||\
+n 9 5
+|\
 n 10 6
-/|)  /N\
+(|\  /||\
 d 12 7
 s 2 3 4 6
-/N\
+/||\
 n 14 8
 |\
 n 15 9
-/|  N\  /N\
+/|  ||\  /||\
 n 16 9
-)N(  M)
+|)  (|\
 d 17 10
-/|\  /N\
+/|\  /||\
 d 19 11
-/N\
+/||\
+n 20 12
+)~|  (|\  ~||\  /||\
 n 21 12
-|)  |\
+|)  (|\
 d 22 13
 s 11
-/|  N\  /N\
+/|  ||\  /||\
 n 23 13
-)N(  M)  (X(
+|)  |\  (|\
 d 24 14
 s 8
-/|\  /N\
+/|\  /||\
+n 25 15
+)~|  /|  ||\  ~||\  /||\
 d 26 15
-/N\
+s 13
+/||\
 d 27 16
-/|  /|)  N\  /N\
+/|  (|\  ||\  /||\
+n 28 16
+|)  |\  (|\
 d 29 17
-/|  N\  /N\
+/|  ||\  /||\
+n 30 18
+)|  /|  (|\  ||\  (||~  /||\
 d 31 18
-/|\  /N\
+/|\  /||\
+n 32 19
+)~|  /|  ||\  ~||\  /||\
+n 33 19
+|)  |\  (|\
 d 34 20
-/|  /|\  N\  /N\
+/|  /|\  ||\  /||\
+n 35 20
+|(  |)  (|\
 d 36 21
 s 9 18
-|)  N)  /N\
+|)  ||)  /||\
+n 37 22
+)|  /|  (|\  ||\  (||~  /||\
 d 38 22
-/|\  /N\
+/|\  /||\
 d 39 23
 s 13
-/|  /|\  (|)  N\  /N\
+/|  /|\  (|)  ||\  /||\
+n 40 23
+|(  |)  (|\
 d 41 24
-/|  /|\  N\  /N\
+/|  /|\  ||\  /||\
+n 42 25
+)|  /|  /|)  (|\  ||\  (||~  /||\
 d 43 25
-|)  N)  /N\
+|)  ||)  /||\
+n 44 26
+)|  /|  (|\  ||\  (||~  /||\
 n 45 26
-/|) /N\
+/|)  /||\
 d 46 27
 s 23
-/|  /|\  (|)  N\  /N\
+/|  /|\  (|)  ||\  /||\
+n 47 27
+|(  |)  |\  (|\
 d 48 28
 s 16

```

Comment:
I assumed not to remove the existing subset notations for the EDOs for which we now have native fifth notations, because of the value of 1.3.9 consistency.

Reply:
That's correct.